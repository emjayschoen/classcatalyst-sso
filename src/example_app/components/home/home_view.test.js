import React from 'react';
import { shallow } from 'enzyme';
import HomeView from './home_view';

describe('<HomeView />', () => {
  let subject;
  beforeEach(() => {
    subject = shallow(<HomeView />);
  });

  it('renders correctly', () => {
    expect(subject).toMatchSnapshot();
  });
});
