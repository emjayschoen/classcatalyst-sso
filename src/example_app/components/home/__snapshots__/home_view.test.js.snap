// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeView /> renders correctly 1`] = `
<styled.div>
  <h3>
    HOME VIEW
  </h3>
</styled.div>
`;
